chart/openfaas/templates/psp.yaml

```diff
@@ -13,27 +13,35 @@ metadata:
         seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
 spec:
     privileged: false
-    hostIPC: false
+    allowPrivilegeEscalation: false
+    requiredDropCapabilities:
+    - ALL
+    volumes:
+    - configMap
+    - downwardAPI
+    - emptyDir
+    - persistentVolumeClaim
+    - projected
+    - secret
     hostNetwork: false
+    hostIPC: false
     hostPID: false
-    readOnlyRootFilesystem: false
-    allowPrivilegeEscalation: false
-    allowedCapabilities:
-        - NET_ADMIN
-        - NET_RAW
-    fsGroup:
-        rule: RunAsAny
-    hostPorts:
-        - max: 65535
-          min: 1
+    readOnlyRootFilesystem: true
     runAsUser:
-        rule: RunAsAny
+        rule: MustRunAsNonRoot
```

Comment:
This breaks Istio and Linkerd support since both of them use an init container that needs root access to setup iptables.

Reply:
As far as I understand, there is nothing preventing people from either setting `psp` to `false` so that this doesn't get created or creating an additional pod security policy that supports Istio/Linkerd.

Reply:
@stefanprodan do you have any further concerns regarding this?